pkgcore/operations/format.py

```diff
@@ -55,7 +56,11 @@ def fetch_one(self, fetchable, observer):
         return True
 
     def failed_fetch(self, fetchable, observer):
-        build_ops = self.domain.build_pkg(self.pkg, observer)
```

Comment:
I'm not sure this line will work. As it passes raw pkg to build_pkg, it will not generate needed build operation from src_operation(build_operations). Seems like it needs PackageWrapper, so that it can call binded _generate_pkg_operations from _ConfiguredTree instead, which gets buildable and run actual nofetch on it.